.github/workflows/playwright.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint, typecheck, build, run and e2e test the CRGA
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Checkout sources
+      uses: actions/checkout@v3
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+
+    - name: Lint and typecheck
+      run: npm test
+
+    - name: Create a build
+      run: npm run build
+
+    - name: Execute CRGA
+      run: >
+        mkdir ./tmp-e2e-app-dir &&
+        npm start tmp-e2e-app-dir
+
+    - name: Run generated app
+      run: >
+        cd tmp-e2e-app-dir &&
+        npm start & sleep 60 &&
+        cd ..
+      timeout-minutes: 3
+
+    - name: Run Playwright tests
+      run:  npm run test:e2e
```

Comment:
Your tests are running on localhost:8080, are you sure you have a localhost at this point?

Reply:
Yes you are right! My bad, i did not see the step `npm start`